sdk/tables/azure-data-tables/inc/azure/data/tables/account_sas_builder.hpp

```diff
@@ -107,14 +107,14 @@ namespace Azure { namespace Data { namespace Tables { namespace Sas {
     Delete = 4,
 
     /**
-     * @brief Indicates that Add is permitted.
+     * @brief Indicates that List is permitted.
      */
-    Add = 8,
+    List = 8,
 
     /**
-     * @brief Indicates that List is permitted.
+     * @brief Indicates that Add is permitted.
      */
-    List = 16,
+    Add = 16,

```

Comment:
Thanks, this fixes part 1 of the issue, which is the order consistency compared to other languages (and the order within the implementation matches the public surface area which is a win from a maintenance perspective):
https://github.com/Azure/azure-sdk-for-cpp/issues/6170

To close the issue, we need to validate that we are only exposing the values needed by the Tables service (i.e. part 2).